openerp/addons/base/i18n/base.pot

```diff
@@ -92,6 +112,14 @@ msgid "\n"
 msgstr ""
 
 #. module: base
+#: code:addons/model.py:146
```

Comment:
... and potentially with an invalid path separator ... on windows :sob:  the same term would produce the same path with a backslash instead of a slash

Do not hesitate to have a look at https://github.com/odoo/odoo/pull/7561, it tries to fix it